### PR TITLE
// 404 page

### PR DESCRIPTION
**Pull request by ishcherbakov**

### CODE REVIEW

File: themes/classic/assets/js/theme.js [hunk 1]
```diff
@@ -9191,7 +9191,7 @@
 /***/ function(module, exports, __webpack_require__) {
 
 	var __WEBPACK_AMD_DEFINE_FACTORY__, __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_DEFINE_RESULT__;/*!
-	 * Datepicker for Bootstrap v1.6.0 (https://github.com/eternicode/bootstrap-datepicker)
+	 * Datepicker for Bootstrap v1.5.1 (https://github.com/eternicode/bootstrap-datepicker)
```
Comment (maximebiloe) [inline themes/classic/assets/js/theme.js]: Are you downgrading the datepicker version ?

File: themes/classic/templates/errors/404.tpl [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {extends file='page.tpl'}
 
 {block name='page_title'}
-  {l s='This page is not available'}
+  {l s='The page you are looking for was not found.'}
```
Comment (djfm) [inline themes/classic/templates/errors/404.tpl]: This is not the page you are looking for

File: themes/classic/assets/js/theme.js (hunk 1 above)
Comment (djfm) [inline themes/classic/assets/js/theme.js]: This change doesn't seem related to the commit, I don't see anything about the datepicker in the source files. Maybe someone forgot to re-build the assets in a parent commit.

**Comment by djfm** [comment]
Much better :)

File: themes/classic/assets/js/theme.js (hunk 1 above)
Comment (ishcherbakov) [inline themes/classic/assets/js/theme.js]: Well. As we use "^x.y.z" syntax in package.json file each person potentially has a different set of external components. So we need to launch "npm update" from time to time and make sure we have latest version before compiling assets for PR.
